package-lock.json

```diff
@@ -34717,7 +34717,7 @@
     },
     "plugins/node/opentelemetry-instrumentation-bunyan/examples": {
       "name": "bunyan-example",
-      "version": "0.45.1",
+      "version": "0.1.0",
```

Comment:
That the ".../examples" entries include a version change is a side-effect of my mistakenly not having updated the package-lock.json file in https://github.com/open-telemetry/opentelemetry-js-contrib/pull/1939.

There is no harm in including these package-lock changes with this PR. However, if we like we could wait for https://github.com/open-telemetry/opentelemetry-js-contrib/pull/1938 to go in -- it removes these ".../examples" dirs from the workspace.

Reply:
All fixed in the current version of this PR. :+1: